AppCenterData/AppCenterDataTests/MSDocumentUtilsTests.m

```diff
@@ -152,7 +111,7 @@ - (void)testDocumentWrapperFromDictionaryWithDocument {
   XCTAssertTrue([[document eTag] isEqualToString:@"etag"]);
   XCTAssertNotNil([document lastUpdatedDate]);
   XCTAssertTrue([[document partition] isEqualToString:@"readonly"]);
-  XCTAssertNotNil([document jsonValue]);
+  XCTAssertNil([document jsonValue]);
```

Comment:
Why should it be `nil`? Was the test wrong before?

Reply:
Yeah now there is no metadata in the json value, just the document itself. It was inconsistent before.

Reply:
This is nil on document parsing error now. Before you could have deserialized value non nil and failed jsonValue being nil.